common/appointment/create.ts

```diff
@@ -35,48 +36,64 @@ export abstract class AppointmentCreateGroupInput extends AppointmentCreateInput
     appointmentType: 'group';
 }
 
+export const isAppointmentOneWeekLater = (appointmentDate: Date) => {
+    const now = moment.now();
+    const start = moment(appointmentDate);
+    const diffDays = start.diff(now, 'days');
+    return diffDays >= 6 ? true : false;
+};
+
+export const isAppointmentFiveMinutesLater = (appointmentDate: Date) => {
+    const now = moment.now();
+    const start = moment(appointmentDate);
+    const fiveMinutesFromNow = moment().add(5, 'minutes');
+    const isSameDayAndFiveMinLater = start.isSame(now, 'day') && start.isAfter(fiveMinutesFromNow);
```

Comment:
Any special reason why it has to be on the same day?

Reply:
because a match appointment can now also be created for now + 5 minutes 😄 but if the date is not today, than I don't need to check if the time ist 5 minutes later than now

Reply:
Well this could just be start > fiveMinutesFromNow, if you compare the whole date instead of manually looking at minutes and days ...